processor/eval/ledger_for_evaluator_test.go

```diff
@@ -101,8 +101,7 @@ func TestLedgerForEvaluatorAsset(t *testing.T) {
 	err = pr.Process(&rawBlock)
 	assert.Nil(t, err)
 
-	ld := indxLeder.MakeLedgerForEvaluator(l)
-	require.NoError(t, err)
```

Comment:
this `err` and similar throughout this file, is being asserted 3 lines above. Hence, I'm removing this redundant assertion.